fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/LibSumoAmbassador.java

```diff
@@ -34,7 +34,7 @@
  */
 public class LibSumoAmbassador extends SumoAmbassador {
 
-    private static final String VALID_LIBSUMO_VERSIONS = "1\\.1[456]\\.";
+    private static final String VALID_LIBSUMO_VERSIONS = "1\\.1[67]\\.";
```

Comment:
What happens if someone wants to use MOSAIC using SUMO 1.15.0 or lower together with libsumo?

Reply:
Error message :) I don't think its necessary to support the older versions.

Reply:
But speaking of Libsumo, I should test it at least locally with 1.17.0, so please don't merge yet.

Reply:
I just assumed you would've done that. 😛 Let me know when this is ready to merge.

Reply:
Done, everything fine :)